vrealize_operations_integration_sdk/stats.py

```diff
@@ -54,16 +54,21 @@ def get_average(inputs: list):
     return sum(inputs) / len(inputs)
 
 
-def get_growth_rate(inputs: list):
-    present = inputs[-1]
-    past = inputs[0]
-
+# TODO: change signature to initial, final, and time
+def get_growth_rate(initial, final, time):
+    """
+    Calculates average percent growth rate per hour
+    :param initial: the initial number of objects
+    :param final: the final number of objects / things
+    :param time: the total time period in seconds
+    :return: float that represents the percent of g
```

Comment:
time is specified as seconds, but description says growth rate per hour -- I don't see any conversion between them, though.

I'd recommend changing the `time` parameter to include the unit, e.g., `get_growth_rate(initial, final, duration_in_seconds)` if seconds is correct.